source/fundamentals/serialization/poco.txt

```diff
@@ -234,6 +594,8 @@ serialization-related attributes:
 
 - ``[BsonElement()]``, which specifies custom field names in the camel case naming convention
 - ``[BsonRepresentation()]``, which specifies serialization of the ``Price`` field as a BSON ``Double`` type
+- ``[BsonDefaultValue("Generic item")]``, which sets the ``Name`` property to
+  ``"Generic item"`` if no value has been assigned to it
```

Comment:
S: as the other bullet items dont include the argument, you can omit here as well and just list the attribute name
```suggestion
- ``[BsonDefaultValue()]``, which sets the ``Name`` property to
  ``"Generic item"`` if no value has been assigned to it
```